pkg/odo/cli/component/common_link.go

```diff
@@ -310,7 +313,7 @@ func (o *commonLinkOptions) completeForOperator() (err error) {
 	}
 
 	// make this deployment the owner of the link we're creating so that link gets deleted upon doing "odo delete"
-	ownerReference := generator.GetOwnerReference(deployment)
+	//ownerReference := generator.GetOwnerReference(deployment)
```

Comment:
Please remove this comment and the others below

Reply:
This is a work in progress, I still need these comments

Reply:
OK